lib/dialects/mysql/connection-manager.js

```diff
@@ -69,7 +69,8 @@ ConnectionManager.prototype.connect = function(config) {
 
         return;
       }
-
+	if (config.onConnect)
+	    config.onConnect(connection);
```

Comment:
Use brackets please. We mostly use single line if's for short-circuits.
There's also an indentation issue.